tracer/src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -366,6 +376,8 @@ private static string ParseString<T>(T headers, string headerName)
             {
                 if (!string.IsNullOrEmpty(headerValue))
                 {
+                    Log.Information("In Parser: " + headerName + " = " + headerValue);
```

Comment:
I don't think we should add this log, as it will be called a log. If we do, we should guard it with Log.IsEnabled, and use structured logging instead of string concat
```suggestion
```

Reply:
Oops. I'm a big fan of adding logs when running the tests. And I often forget some :/ 
Clearly useless line, thanks for spotting